### DISCUSSION TITLE
How to remove Shared Preferences value

### DISCUSSION THREAD

**Discussion by arupnaskarbkp** [bhoominn/nb_utils#5] [Q&A] [answered]
Please Explain How to remove Shared Preferences value

**Comment by bhoominn** [comment, marked as answer]
You can use removeKey method to remove shared preference value.

``` dart
await removeKey('key');
```

**Comment by arupnaskarbkp** [comment]
> You can use removeKey method to remove shared preference value.
> 
> ```dart
> await removeKey('key');
> ```

Thanks Sir
